returns/primitives/container.py

```diff
@@ -66,6 +66,12 @@ def __setstate__(self, state: Union[_PickleState, Any]) -> None:
             # backward compatibility with 0.19.0 and earlier
             object.__setattr__(self, '_inner_value', state)  # noqa: WPS609
 
+    def __replace__(self, **changes: Any) -> 'BaseContainer':
+        """Create a new instance with specified changes."""
+        if set(changes.keys()) - set(self.__slots__):
```

Comment:
I think that we can simplify this to only allow `_inner_value` change. Nothing else.